environments/development/allinone/group_vars/allinone.yml

```diff
@@ -177,7 +177,7 @@ eus_smtp_security: false
 eus_smtp_from_address: yoda@yoda.test
 eus_smtp_replyto_address: yoda@yoda.test
 eus_mail_template: uu
-external_users_domain_filter: uu.nl     # Domains to filter, separated by |
+external_users_domain_filter: uu.nl | acc.uu.nl  # Domains to filter, separated by |
```

Comment:
If the domain filter value allows for whitespace, I think we should also adjust the code template for the is-user-external PAM script to remove any whitespace before using it for regex matching